wake/development/utils.py

```diff
@@ -1653,9 +1653,17 @@ def get_info_from_explorer(
 
     try:
         return _get_sourcify_info(addr, chain_id, cache_dir), "sourcify"
-    except Exception:
```

Comment:
Exception was intentionally there, for example, for the case when Sourcify suffers a temporary outage